diff -up kdsoap-2.0.0/kdwsdl2cpp/libkode/code_generation/printer.cpp.omv~ kdsoap-2.0.0/kdwsdl2cpp/libkode/code_generation/printer.cpp
--- kdsoap-2.0.0/kdwsdl2cpp/libkode/code_generation/printer.cpp.omv~	2022-08-19 18:34:08.185838049 +0200
+++ kdsoap-2.0.0/kdwsdl2cpp/libkode/code_generation/printer.cpp	2022-08-19 18:34:53.260907394 +0200
@@ -762,7 +762,7 @@ void Printer::printHeader(const File &fi
             out.indent();
         }
 
-        if (!clas.isNull()) {
+        if (!clas.isEmpty()) {
             const bool isQtClass =
                     clas.startsWith(QLatin1Char('Q')) && !clas.contains(QLatin1Char('_'));
             if (isQtClass)
